src/parser/bison_parser.y

```diff
@@ -63,6 +63,8 @@
 // %output  "bison_parser.cpp"
 // %defines "bison_parser.h"
 
+%expect 0
```

Comment:
What does this do?

Reply:
https://www.gnu.org/software/bison/manual/html_node/Expect-Decl.html

In other words, bails out with an error if there's a shift/reduce conflict.